File: ex_factorial.c
#include <stdio.h>

int factorial(int n) {
  if (n == 0) { return 1; }
  return n * factorial(n-1);
}

int main() {
  int x = 5;
  int x_fact = factorial(5);
  printf("%d! = %d\n", x, x_fact);
}

File: ex_factorial.py
def factorial(n):
  if(n==0):
    return 1
    
  return n*factorial(n-1)
  
if __name__ == "__main__":
  x=5
  x_fact=factorial(5)
  print(f"{x}! = {x_fact}\n", end="")
  
